modules/gitops-release-notes-1-13-2.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-13-2_{context}"]
+= Release notes for {gitops-title} 1.13.2
+
+{gitops-title} 1.13.2 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-13.2_{context}"]
+== Errata updates
+
+[id="rhsa-2024:8581-gitops-1-13-2-security-update-advisory_{context}"]
+=== RHSA-2024:8581 - {gitops-title} 1.13.2 security update advisory
+
+Issued: 2024-10-29
```

Comment:
How do I verify this date and the advisory number?

Reply:
Do you have an engineering ticket?

Reply:
Hi @dfitzmau - I just talked to my SME @anandf and he had this to reply:

"This may not be needed for Layered products in my opinion. We had such a ticketing process earlier for uploading binaries (kam and argocd-cli), but that is now happening automatically. They had done some automated processes for that." 

I hope this can assuage any fears about this comment.